pennylane/transforms/decompositions/clifford_t_transform.py

```diff
@@ -64,12 +65,26 @@
 _CLIFFORD_T_GATES = tuple(_CLIFFORD_T_ONE_GATES + _CLIFFORD_T_TWO_GATES) + (qml.GlobalPhase,)
 
 # Gates to be skipped during decomposition
-_SKIP_OP_TYPES = (qml.Barrier, qml.Snapshot, qml.WireCut)
+_SKIP_OP_TYPES = (qml.Barrier, qml.Snapshot, qml.WireCut, MeasurementValue)
 
 # Stores the cache of a specified size for the decomposition function
 # that is used to decompose the RZ gates in the Clifford+T basis.
 _CLIFFORD_T_CACHE = None
 
+_CATALYST_SKIP_OP_TYPES = ()
+
+
+# pylint: disable=import-outside-toplevel, global-statement
+def _add_catalyst_skip_op_types():
+    """Delayed addition of PennyLane-Catalyst skip op types."""
```

Comment:
Delayed addition helps with resolving the circular dependency that is encountered while importing `catalyst`, which tries to import `transforms` within its import recipe.